DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: magpie4
 Type: Package
 Title: MAgPIE outputs R package for MAgPIE version 4.x
-Version: 1.86.7
-Date: 2021-02-16
+Version: 1.87.0
+Date: 2021-02-18
 Authors@R: c(person("Benjamin Leon", "Bodirsky", email = "bodirsky@pik-potsdam.de", role = c("aut","cre")),
```

Comment:
don't forget to add your name to the list of authors (forgot to mention that in mrcommons as well)

Reply:
also add your name in the author information of the functions to modify